src/extract_data/get_data_from_csv.py

```diff
@@ -0,0 +1,88 @@
+import numpy as np
```

Comment:
yeah, I have extracted to data getter to a class which I have subbed in to my code now.